DNN Platform/Library/Services/Upgrade/Upgrade.cs

```diff
@@ -4062,65 +4062,22 @@ public static void DeleteInstallerFiles()
         /// -----------------------------------------------------------------------------
         public static string DeleteFiles(string providerPath, Version version, bool writeFeedback)
         {
-            DnnInstallLogger.InstallLogInfo(Localization.Localization.GetString("LogStart", Localization.Localization.GlobalResourceFile) + "DeleteFiles:" + Globals.FormatVersion(version));
-            string exceptions = "";
-            if (writeFeedback)
-            {
-                HtmlUtils.WriteFeedback(HttpContext.Current.Response, 2, "Cleaning Up Files: " + Globals.FormatVersion(version));
-            }
-
-            try
-            {
-                string listFile = Globals.InstallMapPath + "Cleanup\\" + GetStringVersion(version) + ".txt";
-
-                if (File.Exists(listFile))
-                {
-                    exceptions = FileSystemUtils.DeleteFiles(FileSystemUtils.ReadFile(listFile).Split('\r', '\n'));
-                }
-            }
-            catch (Exception ex)
-            {
-                Logger.Error(ex);
-
-                exceptions += string.Format("Error: {0}{1}", ex.Message + ex.StackTrace, Environment.NewLine);
-                // log the results
-                DnnInstallLogger.InstallLogError(exceptions);
-                try
-                {
-                    using (StreamWriter streamWriter = File.CreateText(providerPath + Globals.FormatVersion(version) + "_Config.log"))
-                    {
-                        streamWriter.WriteLine(exceptions);
-                        streamWriter.Close();
-                    }
-                }
-                catch (Exception exc)
-                {
-                    Logger.Error(exc);
-                }
-            }
-
-            if (writeFeedback)
+            var stringVersion = GetStringVersion(version);

```

Comment:
I think we should use new method or overloaded version of this method (or may be just optional parameters) so that there we don't have the check revision and add it separately at each individual usage. Moreover, someone doing some implementation in future might miss that revision check if it is to be done separately. It would be easy for him to compare the two overloads and use them accordingly.